proposed/security-reporting-process-meta.md

```diff
@@ -45,7 +45,7 @@ for how security vulnerabilities go from discovery to fixing to public
 disclosure. Inspiration could be drawn from this list of security disclosure
 processes in various PHP and non-PHP projects:
 
-* http://symfony.com/doc/current/contributing/code/security.html
+* http://symfony.com/security
```

Comment:
What about using HTTPS here instead?

Reply:
We should use HTTPS on all those links, so we should address it with a separate PR

Reply:
see #1048